frontend/src/features/admin-form/common/queries.ts

```diff
@@ -71,11 +71,22 @@ export const useAdminFormCollaborators = () => {
     [form, user],
   )
 
+  const canEditCollaborators = useMemo(() => {
+    if (!form || !user) return false
+    return (
+      user.email === form.admin.email ||
+      form.permissionList.some(
+        (perms) => perms.write && perms.email === user.email,
+      )
```

Comment:
Please use collaborators

Reply:
edb90bd

Reply:
Reversed it and decided to use both as sources of truth because one can load whilst the other is pending: [c6e9392](https://github.com/opengovsg/FormSG/pull/3793/commits/c6e9392963d77da004185d061f8de4871643a1a4)